great_expectations/expectations/expectation.py

```diff
@@ -1999,14 +2079,14 @@ class QueryExpectation(TableExpectation, ABC):
         "condition_parser": None,
     }
 
-    domain_keys = (
+    domain_keys = (  # type: ignore[assignment]
```

Comment:
What's the detailed warning here? 🤔

Reply:
See my write-up in Confluence or we can discuss offline.